app/src/main/java/programatorus/client/transport/wrapper/Transport.kt

```diff
@@ -1,12 +1,11 @@
-package programus.client.transport.wrapper
```

Comment:
Lot of mess with null-handling is this auto-translated from java? Consider refactor